### ISSUE TITLE
User should be informed of bad search parameters

### ISSUE THREAD

**Issue by algorys** [Alignak-monitoring-contrib/alignak-webui#35]
Hello @mohierf ,

If we make this kind of search on webui:

`http://94.76.229.155:91/hosts_table?search=type:host%20state:UP` or `http://94.76.229.155:91/services_table?search=type:host%20state:OK`, this seems broke display of webui.

After that `/hosts_table` and `/services_table` url are not yet available. 

After log out / log in again, problem persist.

**Comment by mohierf** [comment]
Indeed, the display is not broken ... you got 0 elements because your search request did not found anything ! In hosts_table, you do not have any _type_ nor _state_ field ... those fields only exist in the `livestate` table!

You can retrieve a correct display in the table if you click on the clear filter button.

It would be better if the unknown fields in the search were excluded ... I tag this as an enhancement and I rename this issue!
